spark/src/test/java/org/apache/iceberg/spark/source/TestTables.java

```diff
@@ -37,7 +37,7 @@
 import org.apache.iceberg.io.InputFile;
 import org.apache.iceberg.io.LocationProvider;
 import org.apache.iceberg.io.OutputFile;
-import parquet.Preconditions;
+import org.apache.parquet.Preconditions;
```

Comment:
This should not be using Preconditions from Parquet. It should be the guava one. Can you update it to that one instead?

Reply:
Yes.